### PR TITLE
Finished Update Players view functionality.

### PR DESCRIPTION
**Pull request by angadkalra**
@ELCHILEN0 its messy code but I can refactor later. Let me know if anything doesn't make sense. One thing we still gotta do is display error on front end when user enters team ID and that ID doesn't exist.

### CODE REVIEW

File: routes/playerAPI.js [hunk 1]
```diff
@@ -54,4 +54,47 @@ router.get('/:attributes/:conditions/:operators/:constraints/:logic', function(r
     });
 });
 
+router.get('/update/:teamID/:playerID/:jerseyNum/:position/:suspended/:injured', function(req, res, next) {
+  var teamID = (req.params["teamID"] == "tid:") ? "" : req.params["teamID"].substr(4);
+  var jerseyNum = (req.params["jerseyNum"] == "jn:") ? "" : req.params["jerseyNum"].substr(3);
+  var position = (req.params["position"] == "pos:") ? "" : req.params["position"].substr(4);
+  var suspended = (req.params["suspended"] == "sus:") ? "" : req.params["suspended"].substr(4);
+  var injured = (req.params["injured"] == "inj:") ? "" : req.params["injured"].substr(4);
```
Comment (ELCHILEN0) [inline routes/playerAPI.js]: Why take the substring?

Comment (angadkalra) [inline routes/playerAPI.js]: prefix is pos: or tid: etc... so just want the value

Comment (angadkalra) [inline routes/playerAPI.js]: i could do everything efficiently if I had more time. This works and is relatively clean code so I thought we should move on to other parts.

Comment (ELCHILEN0) [inline routes/playerAPI.js]: Ok sounds good, lets merge

**Pull request merged**